src/config.rs

```diff
@@ -99,9 +99,9 @@ impl Config {
             uda_selection_blink: Self::get_uda_selection_blink(),
             uda_calendar_months_per_row: Self::get_uda_months_per_row(),
             uda_style_calendar_title: Self::get_uda_style("calendar.title")
-                .unwrap_or(TColor::new(Color::Black, Color::Rgb(220, 220, 220), vec![])),
+                .unwrap_or(TColor::new(Color::Reset, Color::Reset, vec![])),
             uda_style_context_active: Self::get_uda_style("context.active")
-                .unwrap_or(TColor::new(Color::Black, Color::Rgb(220, 220, 220), vec![])),
+                .unwrap_or(TColor::new(Color::Reset, Color::Reset, vec![])),
```

Comment:
Why change the second argument here?

Reply:
If you mean "why alter `context.active`?":

> It improves readability for example when I have `uda.taskwarrior-tui.style.context.active=blue`.
> ![image](https://user-images.githubusercontent.com/1522460/99197896-66c54680-27e9-11eb-9b74-690ea0202965.png)

If you mean "why replace `Color::Rgb(220, 220, 220)`?"

> It's so that the background comes through from the terminal by default. _I believe_ that `Color::Reset` functions as essentially a "transparent" value when used as a background, and when used as a foreground it applies whatever the primary text colour of a terminal is (typically dark text in a light theme / light text in a dark theme).
>
> This makes sense for a default value as you're essentially removing all the colour formatting and allowing the terminal to define a _generally readable_ default.

Hopefully this answers your question, let me know if I've completely misinterpreted.

Reply:
I was replying from my phone and misread the diff! This is great!